java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -463,6 +465,18 @@ public CompletableFuture<String> getdel(@NonNull String key) {
                 GetDel, new String[] {key}, this::handleStringOrNullResponse);
     }
 
+    @Override
+    public CompletableFuture<String> getex(@NonNull String key) {
+        return commandManager.submitNewCommand(
+                GetEx, new String[] {key}, this::handleStringOrNullResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> getex(@NonNull String key, @NonNull GetExOptions options) {
+        String[] arguments = ArrayUtils.addAll(new String[] {key}, options.toArgs());
```

Comment:
ArrayUtils.addFirst allows you to add a single value to an array and is slightly more performant